### PR TITLE
Add tests for notification by id

### PR DESCRIPTION
**Pull request by m1guelpf**
Add tests for #489

### CODE REVIEW

File: test/Github/Tests/Api/NotificationTest.php [hunk 1]
```diff
@@ -96,6 +96,16 @@ public function shouldMarkNotificationsAsReadForGivenDate()
 
         $api->markRead($since);
     }
+    public function shouldGetNotification()
+    {
+        $id = id;
```
Comment (m1guelpf) [inline test/Github/Tests/Api/NotificationTest.php]: How do I get a ramdom id there?

Comment (sergeyklay) [inline test/Github/Tests/Api/NotificationTest.php]:
```php
mt_rand(1, time()); 
```

:)

**Comment by m1guelpf** [comment]
@sergeyklay Would this work?

File: test/Github/Tests/Api/NotificationTest.php [hunk 2]
```diff
@@ -96,6 +96,17 @@ public function shouldMarkNotificationsAsReadForGivenDate()
 
         $api->markRead($since);
     }
+    public function shouldGetNotification()
+    {
+        $id = mt_rand(1, time());
+        ;
```
Comment (Nyholm) [inline test/Github/Tests/Api/NotificationTest.php]: Remove extra semicolon.

File: test/Github/Tests/Api/NotificationTest.php [hunk 3]
```diff
@@ -96,6 +96,17 @@ public function shouldMarkNotificationsAsReadForGivenDate()
 
         $api->markRead($since);
     }
+    public function shouldGetNotification()
```
Comment (Nyholm) [inline test/Github/Tests/Api/NotificationTest.php]: Add blank line between functions

**Comment by m1guelpf** [comment]
@nyholm Done

**Comment by Nyholm** [comment]
Awesome. Thanks!

**Pull request merged**